XamlControlsGallery/ControlPagesSampleCode/ItemsRepeater/ItemsRepeaterSample1_xaml.txt

```diff
@@ -0,0 +1,44 @@
+<!-- XAML Code -->
+
+<!-- This is the ItemsRepeater used in this example: -->
+<ItemsRepeater x:Name="MixedTypeRepeater" 
+               Margin="0,0,12,0"
+               HorizontalAlignment="Stretch"
+               ItemTemplate="{StaticResource StringOrIntTemplateSelector}">
+
+    <ItemsRepeater.Layout>
+        <UniformGridLayout MinItemWidth="200" MinItemHeight="200"></muxcontrols:UniformGridLayout>
```

Comment:
Just realized that, for consistency, all mentions in sample code of ItemsRepeater or UniformGridLayout should have the muxc namespace prefix. Probably also StackLayout - anything that's defined in WinUI.

```suggestion
        <muxc:UniformGridLayout MinItemWidth="200" MinItemHeight="200"/>
```

Reply:
Fixed all.